e2e/helpers.ts

```diff
@@ -19,6 +19,10 @@ type WritableMetricsResult = {
   -readonly [K in keyof MetricsResult]: MetricsResult[K];
 };
 
+export function logFileContent(fileName = path.resolve('stryker.log')): Promise<string> {
```

Comment:
This does not log anything :P It reads the file content

Reply:
If this is meant to be read as `logfileContent` I would suggest `readLogFife`

Reply:
Yeah, i'll improve the name